src/Path/Container/Shelves.jsx

```diff
@@ -181,7 +181,7 @@ function Shelves() {
                                                         className="px-6 py-4 text-center font-bold"
                                                         style={{ fontFamily: "Poppins, sans-serif" }}
                                                     >
-                                                        {shelve.productId}
+                                                        {shelve.product?shelve.product.title: " "}
```

Comment:
Do it like this

{shelve.products.map(productItem => (
                                                            <div key={productItem.product.id} className="font-bold text-gray-500 truncate" style={{ fontFamily: 'Poppins, sans-serif' }}>
                                                                {productItem.product.title}
                                                            </div>
                                                        ))}
                                                        
                                                        
                                                        Once the products column is created